### DISCUSSION TITLE
How to make only the background transparent, not the whole transparent  when use windowrulev2

### DISCUSSION THREAD

**Discussion by DreamMaoMao** [hyprwm/Hyprland#2837] [General]
when I use this window rule, it makes the background, font, and all UI elements transparent. How can I make only the background transparent?

windowrulev2 = opacity 0.60 0.50,class:^(Code)$

![image](https://github.com/hyprwm/Hyprland/assets/30348075/0385cfa3-0cf0-4b46-8c1b-d08ad3a2a3a8)

**Comment by 5urajkharkwal** [comment]
same
